application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -488,6 +491,18 @@ type Emissions {
   co2: Grams
 }
 
+"Station entrance or exit, originating from OSM or GTFS data."
+type Entrance {
+  "Short text or a number that identifies the entrance or exit for passengers. For example, `A` or `B`."
+  code: String
```

Comment:
I think @t2gran wanted this to be called publicCode, but I don't have a strong preference for either way. The public doesn't really make it more understandable for me but I'm used to the stop code concept.